08_voting/index.tex

```diff
@@ -1 +1,192 @@
+
+\textbf{Changes of rules:} the President can select a rule from the proposed list to be voted by the Voter islands. The Speaker will host the voting for changes of rules and be responsible for the ballots counting and results declaration. The ballots from the Voter islands will be: Aye, namely agree with the rule change; Nay, namely disagree with the rule change; or Abstain.
+
+
+\section{Voting Methods}
+\label{sec:VotingMethods}
+
+Table \ref{table:votingmethod} indicates available choices for each voting scenario.
+
+\begin{table}[H]
+\caption{Available voting methods for each voting scenario}\label{table:votingmethod}
+\begin{center}
+\begin{tabular}{ |p{3cm}||p{3cm}|p{3cm}|  }
+ \hline
+ Voting Methods   & Elections of Roles & Changes of Rules   \\
+ \hline
+ Plurality   &     & \checkmark     \\
+ \hline
+ Borda Count &  \checkmark   &      \\
+ \hline
+ Runoff      &  \checkmark    &        \\
+ \hline
+ Instant Runoff    & \checkmark  &     \\
+ \hline
+ Approval  &  \checkmark    &    \\
+ 
+ \hline
+\end{tabular}    
+\end{center}
+\end{table}
+
+\textbf{Plurality} will select the majority choices as the collective decision. When the decision is picking between two options, e.g. whether agree or disagree a change of rule. Unlike the elections, it is not necessary to compare the options left for determining a relative preference. It is straightforward and effective especially when the number of valid ballots (i.e. abstention excluded) is odd. However, it requires consideration when the results are tied. The solution to tied results is regimented with the assumption that the proposer of the voting in such events will vote for "agree". Thus, the proposal will be approved if the number of "agree" votes is more than the number of "disagree" votes. 
+
+\textbf{Borda Count} method will allow voters to rank preference instead of choosing one option only. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all voters' ballots produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the collective decision.
+
+% The tied results are resolved by selecting the most first-nominated candidate. If this still gives multiple winners, the most second-nominated candidate will be selected as the winner. This iterative counting process will act as a tiebreaker for winner selection.
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which is used to pick a single winner from a list of several candidates. 
+%Normally, voters cast a single vote for their preferred candidate, but in our game specification it is also possible for voters to make their preference list.
+Runoff voting is widely recognized and used around the world primarily for the election of the head of state. It is an effective voting method for the selection of candidates.
```

Comment:
give a very specific example. 

around the world is NOT specific enough.

again, plz do NOT say ''effective''

Reply:
rewrited